whelk-core/src/integTest/groovy/MarcFrameConverterSpec.groovy

```diff
@@ -7,9 +7,17 @@ import spock.lang.*
 @Unroll
 class MarcFrameConverterSpec extends Specification {
 
-    static converter = new MarcFrameConverter() {
-        def config
+    static Whelk whelk = null
+    static {
+        try {
+            whelk = Whelk.createLoadedSearchWhelk()
+        } catch (Exception e) {
+            System.err.println("Unable to instantiate whelk: $e")
```

Comment:
Do we also need to warn that some tests may not be carried out?

Reply:
Good suggestion; addressed in 97b471a618bfd25099cdc6235659223d21484966.